src/js/utils/paste-utils.js

```diff
@@ -11,11 +10,14 @@ export function setClipboardCopyData(copyEvent, editor) {
 
   const range = cursor.offsets;
   const mobiledoc = post.cloneRange(range);
+  const fragment = getSelectionContents();
+  const div = createDiv();
+  div.appendChild(fragment);
   const html =
     `<div data-mobiledoc='${JSON.stringify(mobiledoc)}'>` +
-      `<a href='${issueUrl}'>Pasting from Content-Kit not yet supported.</a>` +
+      div.innerHTML +
```

Comment:
I gave this some thought before- and I think it should perhaps copy a `mobiledoc-html-renderer` or `dom-renderer` version of the HTML.

* The `innerHTML` may have cards in an edit state, which would not be what we really want to copy.
* The DOM of the content-kit-editor is specialized, for example having empty `p` tags in a few cases where we need a place for the cursor. These tags are not really part of documents, however.